### PR TITLE
Makefile refactoring

### PR DESCRIPTION
**Pull request by Ubun1**
* create help target
* create lint target
* swap GIT ifdef clause with assert-command function

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,41 +19,54 @@ ifndef TAG
...
+tests: ; @tox
 
-ifdef GIT
-clean-spec:
-	@git checkout $(PACKAGE).spec
-endif
+clean-spec: ## clean spec file
+clean-spec: ; @git checkout $(PACKAGE).spec
 
+clean: ## build artifacts
```
Comment (Ubun1) [inline Makefile]: change comment for target

File: Makefile [hunk 2]
```diff
@@ -18,41 +19,54 @@ ifndef TAG
...
-	@tox
+tests: ## run tests
+tests: ; @tox
 
-ifdef GIT
-clean-spec:
-	@git checkout $(PACKAGE).spec
-endif
+clean-spec: ## clean spec file
+clean-spec: ; @git checkout $(PACKAGE).spec
```
Comment (Ubun1) [inline Makefile]: do not merge targets

File: Makefile [hunk 3]
```diff
@@ -18,41 +19,54 @@ ifndef TAG
...
-spec:
+help: ## show help
+	grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | \
+	sort | \
+	awk 'BEGIN {FS = ":.*?## "}; {printf "%-30s %s\n", $$1, $$2}'
+
+lint: ## lint spec file
+lint: export _check = $(call assert-command-present, rpmlint)
+lint: spec
+lint: ; @rpmlint $(PACKAGE).spec
```
Comment (Ubun1) [inline Makefile]: do not use one-line targets

File: Makefile [hunk 4]
```diff
@@ -18,41 +19,54 @@ ifndef TAG
...
 	          --define "_srcrpmdir $(CURDIR)/srpms" $(PACKAGE).spec
 
+rpm: ## create rpm
 rpm: srpm
 	@mkdir -p rpms/$(DIST)
 	/usr/bin/mock -r $(DIST) --resultdir rpms/$(DIST) \
 	              --rebuild srpms/$(PACKAGE)-$(VERSION)-$(RELEASE).src.rpm \
 
 ifdef TAG
+copr: ## build package in corp repo
```
Comment (pkulev) [inline Makefile]: corp -> copr

File: Makefile [hunk 5]
```diff
@@ -18,41 +19,54 @@ ifndef TAG
...
 copr: srpm
 	@copr-cli build --nowait c2devel/c2-sdk \
 	          srpms/$(PACKAGE)-$(VERSION)-$(RELEASE).src.rpm
 else
+copr: ## build package in corp testing repo
```
Comment (pkulev) [inline Makefile]: corp -> copr

**Pull request merged**